controllers/appwrapper_controller.go

```diff
@@ -223,28 +229,27 @@ func onAdd(obj interface{}) {
 	aw, ok := obj.(*arbv1.AppWrapper)
 	if ok {
 		klog.Infof("Found Appwrapper named %s that has status %v", aw.Name, aw.Status.State)
-		if aw.Status.State == arbv1.AppWrapperStateEnqueued || aw.Status.State == "" {
+		if aw.Status.State == arbv1.AppWrapperStateEnqueued || aw.Status.State == "" && aw.Labels != nil {
```

Comment:
appwrapper can have different kinds of labels, may be we should check if orderedInstance label exists or not

Reply:
Thanks for pointing this out. 
Also that particular check is no longer needed due to #96

So, I think I'll remove that completely, what do you think?

Reply:
InstaScale looks for `"orderedinstance": "test.instance1"` to scale-up, so we should only scale-up when such a label exists inside an appwrapper, can you please confirm if we are doing the same?